### PR TITLE
[#4390] Add per diem details.

### PR DESCRIPTION
**Pull request by lgheorghiu**
Scope
=====

Add details on per diem.

Changes
=======

Inserted the yearly team building and per diem details in the leave.rst file.

How to try and test the changes
===============================

reviewers: @adiroiban @dumol @hcs0 

I am currently unable to test this on Windows environment.  Please check not only the content but the formatting as well.

### CODE REVIEW

**Comment by lgheorghiu** [comment]
needs-review

File: docs/office/leave.rst [hunk 1]
```diff
@@ -74,3 +74,25 @@ Unused leaves
...
+
+During these periods work sessions are alternated with get to know each other
+hangouts.
+
+
+Daily allowence
+===============
+
+While travel and accomodation expences are covered by the
+company, a daily allowence is to be payed for each employee.
```
Comment (hcs0) [inline docs/office/leave.rst]: it should be `a daily allowance is to be paid for each employee`.

Comment (hcs0) [inline docs/office/leave.rst]: it should be `a daily allowance is to be paid for each employee`.

**Comment by adiroiban** [comment]
Look good, but I think that this can be reviewed by Hannah and Misu

File: docs/office/leave.rst [hunk 2]
```diff
@@ -74,3 +74,25 @@ Unused leaves
...
+Meet in person
+==============
+
+In person interaction is important for any team.
+As all team members are working from different places and do not
+get the chance to have the coffee chit chat or the idle work breaks, 
+we try to get the team together at least once a year.
+Usually it's for a week, each time in a different location.
+
+During these periods work sessions are alternated with get to know each other
```
Comment (hcs0) [inline docs/office/leave.rst]: I'm thinking:

`During the team building meeting, work sessions are alternated with hangouts to get to know each other.`

File: docs/office/leave.rst [hunk 3]
```diff
@@ -74,3 +74,25 @@ Unused leaves
   up to a total of 5 days.
   Try to plan your time so that you don't end up
   with these remaining days in the first place.
+
+
+Meet in person
```
Comment (hcs0) [inline docs/office/leave.rst]: Maybe `In-Person Team Building Meeting`

**Comment by hcs0** [comment]
I added some feedback.

changes-approved

**Comment by dumol** [comment]
**changes-approved**

**Pull request merged**